compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -806,7 +806,11 @@ pub trait AttributeExt: Debug {
     /// * `#[doc(...)]` returns `None`.
     fn doc_str_and_comment_kind(&self) -> Option<(Symbol, CommentKind)>;
 
-    fn style(&self) -> AttrStyle;
+    fn style(&self) -> AttrStyle {
+        self.opt_style().unwrap()
+    }
```

Comment:
nit: since we're already touching this code, we can have a better panic message
```suggestion
    fn style(&self) -> AttrStyle {
        match self.opt_style() {
            Some(style) => style,
            None => panic!("AttributeExt::style: `{self:?}` has no style"),
        }
    }
```